digitalpy/core/domain/object_id.py

```diff
@@ -137,4 +137,4 @@ def parse_oid_string(oid: str):
 
         prefix = ObjectId.DELIMITER + oid_parts
 
-        return {"type": type, "id": ids, "prefix": prefix}
+        return {"type": type, "id": ids, "prefix": prefix}
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>W292 no newline at end of file